### DISCUSSION TITLE
Global and regional ocean mass budget closure since 2003

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#19] [Paper discussions]
https://doi.org/10.1038/s41467-024-45726-w

They use my Greenland mass balance http://doi.org/10.5194/essd-13-5001-2021 (M2021) which excludes peripherals, and then use https://doi.org/10.5194/tc-15-3135-2021 for peripheral glaciers (note, the two authors of the latter are included in the former).

Per 2.1.2 they (@bmarzeion and others I cannot  find on GitHub) use RGI 6.0 and exclude CL2 (strongly connected). That removes the Geike Plateau and all red areas in the following figure (generated by me for this issue, not in a publication), but purple areas are RGI 6.0 != CL2, and gray overlay is used in M2021.

![image](https://github.com/IACS-cryo/Delineation-WG/assets/145117/17fbd4d0-00c8-4197-8338-9b956a290a5a)

**Comment by fmaussion** [comment]
M2021'a grid seems quite coarse for the smaller, more complex ice bodies along the coast!

**Reply by mankoff** [reply]
That's the problem with a 5 km RCM grid vs higher resolution data. I address how I handle this in more detail in https://doi.org/10.5194/essd-12-2811-2020

Although looking at it now, those big boxes might be 25 km, not 5 km - in which case that graphic has a bug - but still highlights the issue of continued overlap in papers that claim to 'close budgets'.
